docs/en/operations/named-collections.md

```diff
@@ -2,34 +2,90 @@
 slug: /en/operations/named-collections
 sidebar_position: 69
 sidebar_label: "Named collections"
+title: "Named collections"
 ---
 
-# Storing details for connecting to external sources in configuration files
+Named collections provide a way to store collections of key-value pairs to be 
+used to configure integrations with external sources. You can use named collections with
+dictionaries, tables, table functions, and object storage. 
 
-Details for connecting to external sources (dictionaries, tables, table functions) can be saved
-in configuration files and thus simplify the creation of objects and hide credentials
-from users with only SQL access.
+Named collections can be configured with DDL or in configuration files and are applied
+when ClickHouse starts. They simplify the creation of objects and the hiding of credentials
+from users without administrative access.
 
-Parameters can be set in XML `<format>CSV</format>` and overridden in SQL `, format = 'TSV'`.
-The parameters in SQL can be overridden using format `key` = `value`: `compression_method = 'gzip'`.
+The keys in a named collection must match the parameter names of the corresponding
+function, table engine, database, etc. In the examples below the parameter list is
+linked to for each type.
 
-Named collections are stored in the `config.xml` file of the ClickHouse server in the `<named_collections>` section and are applied when ClickHouse starts.
+Parameters set in a named collection can be overridden in SQL, this is shown in the examples
+below.
+
+## Storing named collections in the system database
+
+### DDL example
+
+```sql
+CREATE NAMED COLLECTION name AS
```

Comment:
I think we should also document `ALTER NAMED COLLECTION .. SET .. DELETE ..` and `DROP NAMED COLLECTION ..`